tutorials/sar-basics/index.md

```diff
@@ -0,0 +1,181 @@
+
+Sentinel-1 can acquire in different **modes**, by programming the sensor to acquire at different resolution or polarization combinations. The most common mode over land is Interferometric Wide (IW), which is acquired in VV and VH polarization (in some maritime and polar areas in HH and HV). Extended Wide (EW) is used over maritime areas. The main trade-off between the 2 modes is between resolution and swath width, i.e. the size of the area that is illuminated by the SAR beam in range direction. The IW mode swath is approximately 250 km in size, where EW mode can cover a 400 km swath (both modes are actually composed of 3-5 subswaths).
+
+Sentinel-1 is **right looking**, i.e. the side looking antenna is sending microwave beams at a 90 degree right angle relative to the sensor's flight path (= **azimuth** direction). As stated before, a SAR collects timed range samples of the backscattered signal of a microwave pulse that is sent with a discrete frequency along the azimuth direction. The first backscattered signal is registered for the swath edge that is closest to the sensor orbit, which is called the **near range**. The last backscattered signal sample is registered at the **far range** edge of the swath. From the sensor view configuration, it is easy to figure out that near range samples have a lower **incidence angle** than far range samples, and that incidence angle varies regularly (though not linearly) over the swath. Furthermore, DESC orbits have their near to far range angles increasing from roughly east to west (rotated with the orbit angle), whereas for the ASC orbit this is roughly west to east. The combination of different incidence and **look angle** can lead to different backscattering depending on the target. 
+
+The concept of an image is only introduced after Level 1 processing, which re-arranges the Level 0 samples in azimuth and range bins to locations on earth. In order to keep Level 1 images at a reasonable size (e.g. for downloading), the Level 0 data is cut into azimuth time slices of 25 seconds (for IW), which are then processed to GRD. This is where the term **frame** or **scene** is often used. 25 seconds of azimuth time corresponds to approximately 185 km om the ground. 
+
+We now have (almost) all relevant parameters to understand how Sentinel-1 views an area of interest. Resolution and pixel spacing are explained in more detail when we deal with speckle. In the next script, we'll highlight some practical aspects of what we just learned.
+
+```
+// The area of interest can be defined by drawing and area in the editor's map tab
```

Comment:
"can be defined drawing and area editor's map tab" -> "can also be defined by drawing a shape in the Code Editor map"

Reply:
revised